src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import frc.robot.Constants.ClimberConstants;
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.kOff;
+
+public class ClimberSubsystem {
+
+    private final DoubleSolenoid m_leftSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, ClimberConstants.leftForwardChannel, ClimberConstants.leftReverseChannel);
+    private final DoubleSolenoid m_rightSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, ClimberConstants.rightForwardChannel, ClimberConstants.rightReverseChannel);
+    
+    private final Compressor m_compressor = new Compressor(PneumaticsModuleType.REVPH);
+    
+    private boolean enableCompressor = true;
+    
+    public ClimberSubsystem(){
+        m_leftSolenoid.set(kOff);
+        m_rightSolenoid.set(kOff);
+        m_compressor.disable();
+        m_compressor.enableAnalog(ClimberConstants.minPressure, ClimberConstants.maxPressure);
+    }
+    
+    //Runs once every tick (~20ms)
+    public void periodic() {
+        // SmartDashboard.putNumber("Pressure", m_compressor.getPressure());
+        // SmartDashboard.putBoolean("Compressor Enabled", m_compressor.isEnabled());
+        // SmartDashboard.putBoolean("Startup Compressor", enableCompressor);
+    }
+    /*
+     * Sets the state of the solenoid to off 
+     */
+    public void off(){
+        m_leftSolenoid.set(kOff);
+        m_rightSolenoid.set(kOff);
+    }
+
+    //TODO: fix doc
+    /*
+     * Extends both arms
+     */
+    public void forward() {
+        m_leftSolenoid.set(kOff);
```

Comment:
Literally all of these methods with different names set the solenoids to kOff. This is another major bug that really should never even have been committed.